src/helpers/ConfigHelper.ts

```diff
@@ -65,7 +65,18 @@ export class ConfigHelper {
 
   public static doesCurrentOriginMatchConfigOrigin(configOrigin: string): boolean {
```

Comment:
We should list the conditions we are checking as a comment here, since this method does a lot more now.